tests/storage/snapshot.go

```diff
@@ -595,18 +600,18 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				_, err = virtClient.VirtualMachineSnapshot(snapshot.Namespace).Create(context.Background(), snapshot, metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
 
-				Eventually(func() bool {
+				Eventually(func() string {
 					updatedVMI, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, &metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
-					return updatedVMI.Status.FSFreezeStatus == "frozen"
-				}, time.Minute, 2*time.Second).Should(BeTrue())
+					return updatedVMI.Status.FSFreezeStatus
+				}, time.Minute, 2*time.Second).Should(Equal("frozen"))
 
 				deleteSnapshot()
-				Eventually(func() bool {
+				Eventually(func() string {
 					updatedVMI, err := virtClient.VirtualMachineInstance(vm.Namespace).Get(context.Background(), vm.Name, &metav1.GetOptions{})
 					Expect(err).ToNot(HaveOccurred())
-					return updatedVMI.Status.FSFreezeStatus == ""
-				}, time.Minute, 2*time.Second).Should(BeTrue())
+					return updatedVMI.Status.FSFreezeStatus
+				}, time.Minute, 2*time.Second).Should(BeEmpty())
```

Comment:
Is this more informative?  Wouldn't `gstruct` fit more here as well?

Reply:
I dont think there is anything in the status struct that will help in this case

Reply:
The `gstruct` will also show what field should be `""`

Reply:
I know, but in this case I dont think getting the entire status will help with the debug, so dont want to return it all